extension.js

```diff
@@ -23,11 +26,9 @@ function init() {
 }
 
 function enable() {
-	let appMenu=Main.panel.statusArea.appMenu.actor.get_parent();
```

Comment:
Comment our these 3 lines instead oof removing them please. For people who want to default to the old style (without implementing a preferences dialog)